jest.config.js

```diff
@@ -21,6 +20,10 @@ module.exports = {
     // Use the client-side logger by default for tests.
     '^core/logger$': '<rootDir>/src/core/client/logger',
     '^core/window$': '<rootDir>/src/core/browserWindow',
+    // Alias tests for tests to be able to import helpers.
+    '^tests/(.*)$': '<rootDir>/tests/$1',
+    // Alias package.json so it can be imported.
+    '^package$': '<rootDir>/package.json',
```

Comment:
Why does `package` need to look like a module? I think this is confusing because when you look at `import package from 'package'` it's not obvious where that comes from. Can't we just do `JSON.parse(readFileSync(...))`?